learn/getting_started/installation.md

```diff
@@ -95,12 +96,14 @@ cargo build --release
 :::
 
 ::: tab Windows
+
 To install MeiliSearch on Windows, you can:
 
 - use Docker
 - [download the latest binary](https://github.com/meilisearch/MeiliSearch/releases)
 - compile from source
-:::
+
+If you are new to the Windows CMD, you can learn more about it [here](https://www.makeuseof.com/tag/a-beginners-guide-to-the-windows-command-line/).
```

Comment:
What do you think about my other comment on dropping the conditional to simplify this line? And do you think we could avoid using `here` as a link text?